saleor/settings.py

```diff
@@ -599,6 +600,7 @@ def get_host():
         "module": "saleor.payment.gateways.braintree",
         "config": {
             "auto_capture": True,
+            "store_card": False,
```

Comment:
I assume that this value should be taken from env first. (Same as `auto_capture` btw)